.travis.yml

```diff
@@ -20,7 +20,7 @@ script:
  - cabal v2-update
  - cabal v2-build
  - cabal v2-test --enable-test
- - cabal new-haddock
+ - "! cabal new-haddock 2>&1 >/dev/null | grep 'Failed to build documentation'"
```

Comment:
Maybe we can add comment here about haskell/cabal#5977 - we probably want to revert this after it gets fixed.